### ISSUE TITLE
Grébaut’s entries

### ISSUE THREAD

**Issue by MagdaKrzyz** [BetaMasaheft/Dillmann#448]
I am sending the list of all entries from Grebaut’s Supplement that I have done,  so that it is not only on my computer. 
[Entries_Grebaut.docx](https://github.com/BetaMasaheft/Dillmann/files/10499659/Entries_Grebaut.docx)

**Comment by eu-genia** [comment]
I think I have now created a separate meaning for Grébaut to add (fr) at the upper level, we can try if this is more convenient, then I would eventually edit the existing records

**Comment by MagdaKrzyz** [comment]
There is a separate meaning for Grebaut and for Leslau now, right? It might be a good idea to have a separate tab for each source but then a bunch of changes will be needed for the entries already done and for the future ones:
1) we can abandon specifying <L< >L>, <G< >G> since this was moved to the upper level
2) we will have to specify the language for the past entries: Latin and French for Gr, English for Les (so far it has been common English for the whole Traces level)
3) one has to manually adjust all the cross references already done because they some of them refer to both Leslau and Gr, some only to Gr or some only to Les.
4) There is a serious limitations of how many sources (Gr, Les, Traces, Other) you may enter because of the language. First you choose the language and only then the source. You can have only  English, French etc once. That means if we have already source: Traces in English it is impossible to have also source: Leslau in English. 
What do you suggest? 
Here a model entry with Les and Gr separately, you may try the languages on it.
https://betamasaheft.eu/Dillmann/lemma/L6061b57242be4aa89c3a2526889ecc24

**Comment by MagdaKrzyz** [comment]
ad. 4 you can have only one source for a given en/fr/de meaning. At least this is what I experience when I want to add more

**Comment by MagdaKrzyz** [comment]
we would need to have a possibility: "add more sources", to each meaning

**Comment by eu-genia** [comment]
It is very complicated as the source uses lang is primary id and it can be only once. I cannot do it, it means a total reprogramming.

**Comment by eu-genia** [comment]
This too means significant revision of the code. Once there is a project with funds for development all should be reprogrammed. Not possible at this stage.

**Comment by eu-genia** [comment]
At some point one should deprecate the web editor and use GitHub workflow and edit XML. Then most of the problems of this type would not exist.

**Comment by MagdaKrzyz** [comment]
Ok, I understand, but in that case why have you introduced the sources: Gr, Les and Other in the interface? It was necessary to fix the bug? If it is not a real choice for the editor, it would be better to remove them and leave only Dillmann and Traces.

**Comment by MagdaKrzyz** [comment]
"At some point one should deprecate the web editor and use GitHub workflow and edit XML." I see, but the work will be even more time-consuming, won't it?

**Comment by eu-genia** [comment]
> "At some point one should deprecate the web editor and use GitHub workflow and edit XML." I see, but the work will be even more time-consuming, won't it?

Don't think so.

**Comment by eu-genia** [comment]
> Ok, I understand, but in that case why have you introduced the sources: Gr, Les and Other in the interface? It was necessary to fix the bug? If it is not a real choice for the editor, it would be better to remove them and leave only Dillmann and Traces.

Was an attempt. The way the app works at the moment it does not let to use the same language twice but it does not limit other languages or checks whether they are real. So using fr and Grebaut works well but also something like "ab" " bc" or "xy" for more.

Anyway will try to solve the subsenses issue, for the moment subsenses G L etc can be added at edit stage but not at create stage.

**Comment by eu-genia** [comment]
Ok, I see there is only TRaces and Other. What is the function of "Other"?

_Originally posted by @MagdaKrzyz in https://github.com/BetaMasaheft/Dillmann/issues/456#issuecomment-1774949470_

**Comment by eu-genia** [comment]
No function at the moment, A placeholder. I removed Grébaut and Leslau as you did not want them.

**Comment by MagdaKrzyz** [comment]
ok
